gym/lib/gym/runner.rb

```diff
@@ -155,14 +164,35 @@ def move_ipa
       ipa_path
     end
 
+    # copys framework from temp folder:
+
+    def extract_framework_files(path)
+      tool_path = Dir[path]
+      tool_path.each do |f|
+        FileUtils.cp_r(f, File.expand_path(Gym.config[:output_directory]), remove_destination: true)
+      end
+      UI.success "Successfully exported the Framework files:"
+      UI.message tool_path.join("\n")
+      tool_path.join("\n")
+    end
+
+    # copys file out of xcarchive
+    def extract_files_from_archive(cmd_path)
+      tool_path = Dir[File.join(BuildCommandGenerator.archive_path, cmd_path)]
```

Comment:
Looks like there is some copy pasta going on with the `tool_path` variable. Should this be something like `archive_path`?